packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  docker,
+  findContainerByIP,
+  dockerComposeUp,
+} from "@dappnode/dockerapi";
+import { logs } from "@dappnode/logger";
+import { params } from "@dappnode/params";
+import { getDockerComposePath } from "@dappnode/utils";
+import Dockerode from "dockerode";
+
+async function disconnectConflictingContainerAndStartBind(): Promise<void> {
+  const network = docker.getNetwork(params.DOCKER_PRIVATE_NETWORK_NAME);
+  const conflictingContainer = await findContainerByIP(network, params.BIND_IP);
+  if (conflictingContainer) {
+    logs.info(
+      `address ${params.BIND_IP} already in used by ${conflictingContainer.Name}, freeing it`
+    );
+    await network.disconnect({ Container: conflictingContainer.Name });
+  }
+  logs.info(`Starting ${params.bindContainerName} container`);
+  // The docker compose will start the container with the right IP
+  await dockerComposeUp(getDockerComposePath(params.bindDnpName, true));
```

Comment:
Here are we 100% sure that `ensureBindComposeIp` has been executed?

Reply:
yes!